package.json

```diff
@@ -10,14 +10,16 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/aremu-smog/termii.git"
+    "url": "git+https://github.com/Fn-studyo/termii.git"
   },
   "keywords": [
     "sms",
+    "otp",
+    "verify",
     "messaging",
     "termii"
   ],
-  "author": "Aremu Oluwagbamila (SMOG)",
+  "author": "Muritala David",
```

Comment:
Instead of overriding, the author name, you can append your name here as a contributor. Kindly see [this](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors) for more information

Reply:
Alright boss